app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find_by_id(params[:user_id])
+    @posts = @user.posts
+  end
+
+  def show
+    @post = Post.find_by_id(params[:id])
+  end
+end
```

Comment:
- For this milestone, you are only required to setup your controllers, hence you do not require any logic in your controller yet. Due to the logic, since your database is not seeded yet, in your `index` controller, I cannot view the index page properly. Please fix this 🙏.

Your project:
![ardgar](https://user-images.githubusercontent.com/73607512/236814931-6a6c8e27-9909-4d2e-b060-7d7bd8f7f867.PNG)